### PR TITLE
hack for cleaning up names in ETS data

### PR DESCRIPTION
**Pull request by malaffoon**
More of an FYI than a review but ...

These are the scripts i used to clean the names in the ETS data (both UAT and QA).

### CODE REVIEW

File: warehouse/sql/clean_names_hack.sql [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+  name      VARCHAR(60),
+  INDEX idx__last_name_mapping__hash_name(hash_name)
+);
+CREATE TABLE IF NOT EXISTS datagen.ssid_mapping (
+  hash_ssid VARCHAR(65),
+  ssid VARCHAR(10)
+);
+
+-- collect first "names" that we want to map
+INSERT INTO datagen.first_name_mapping (hash_name, gender_id)
```
Comment (malaffoon) [inline warehouse/sql/clean_names_hack.sql]: Note that the primary mapping tables preserve both the old and new names so we can revert all this if we really need to.

File: warehouse/sql/clean_names_hack.sql [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+UPDATE datagen.district_name_mapping SET name =
+  CONCAT_WS(' ',
+            (SELECT name FROM datagen.animal_names ORDER BY RAND() limit 1),
+            (SELECT name FROM datagen.animal_names ORDER BY RAND() limit 1),
+            'District')
+  WHERE name IS NULL;
+
+-- make an import record for these changes
+INSERT INTO warehouse.import (status, content, contentType, digest, batch)
+VALUES (0, 4, 'text/plan', 'MANUAL', 'mlaffoon cleaning organization data');
```
Comment (malaffoon) [inline warehouse/sql/clean_names_hack.sql]: Thanks @agorina!

File: warehouse/sql/clean_names_hack.sql (hunk 1 above)
Comment (agorina) [inline warehouse/sql/clean_names_hack.sql]: Nice.

File: warehouse/sql/clean_names_hack.sql (hunk 2 above)
Comment (agorina) [inline warehouse/sql/clean_names_hack.sql]: :)

**Pull request merged**